superset-frontend/src/components/PageHeaderWithActions/index.tsx

```diff
@@ -19,6 +19,7 @@
 import React, { ReactNode, ReactElement } from 'react';
 import { css, SupersetTheme, t, useTheme } from '@superset-ui/core';
 import { AntdDropdown, AntdDropdownProps } from 'src/components';
+import { TooltipPlacement } from 'antd/es/tooltip';
```

Comment:
Would you mind exporting this from the Tooltip component in our components library and then importing it here? I think we should centralize all the components related things and avoid accessing antd directly

Reply:
Good thinking! Fixed in [`this commit`](https://github.com/apache/superset/pull/21189/commits/890af00638acaa42d9f7d35bd86f3ad573739122).